action.go

```diff
@@ -343,7 +343,7 @@ func (a Action) String() (str string, err error) {
 // permission is found, the default one `default` is used.
 // The first permission that is found to apply to an operation, but
 // doesn't allow the operation to run, will fail the verification globally
-func (a Action) VerifyACL(acl ACL, keyring io.Reader) (err error) {
+func (a Action) VerifyACL(acl ACL, keyring io.Reader, verifyACLs bool) (err error) {
```

Comment:
Sorry just realized the last commit I failed to update the actual call to the VerifyAcls function. [d177bd8](https://github.com/jboyer2012/mig/commit/d177bd86dc821d70fe5a1a570d0b478f8bb0dda8) fixed it.